app/models/rules_of_origin/data_set.rb

```diff
@@ -15,7 +15,7 @@ def load_default
     private
 
       def default_scheme_set
-        RulesOfOrigin::SchemeSet.from_default_file(TradeTariffBackend.service)
+        RulesOfOrigin::TradingScheme.from_default_file(TradeTariffBackend.service)
```

Comment:
Is this reflective of the name Matt has chose for scheme of schemes or is this our choice of name?

Reply:
this is my choice, I changed it since Scheme is very generic,
and the proper name is Trading Scheme (see https://www.gov.uk/guidance/understanding-rules-of-origin-under-the-developing-countries-trading-scheme)
but, I realized I removed the "Set" too, which has actually meaning since there is not just a trading scheme

What about renaming it to `TradingSchemeSet`?

Reply:
we've clarified this point. No need to rename it.